doc/userguide/lua/libs/flowvar.rst

```diff
@@ -0,0 +1,79 @@
+Flowvar
+#######
+
+The ``suricata.flowvar`` library exposes flow variables to Lua
+scripts.
+
+Initialization
+--------------
+
+First, the ``flowvar`` lib must be required::
```

Comment:
nit: while I understand this, the wording feels a bit weird.
Suggestion:
```suggestion
First, the ``flowvar`` lib must be indicated as required::
```

Reply:
How about:

> First, the ``flowvar`` module must be loaded::